.github/workflows/package-apisix-rpm-ubi.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
       - name: run ubi8 docker and mapping rpm into container
         run: |
-          docker run -itd -v $PWD/output:/output --name ubiInstance --net="host" registry.access.redhat.com/ubi8/ubi:8.6 /bin/bash
+          docker run -itd -v $PWD/output:/output -v $(pwd)/test/apisix/config.yaml:/usr/local/apisix/conf/config.yaml  --name ubiInstance --net="host" registry.access.redhat.com/ubi8/ubi:8.6 /bin/bash
```

Comment:
config.yaml contains admin_key configuration.
APISIX master has a new change that would automatically regenerate a new admin_key thereby making it difficult to continue tests in the CI.